atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -374,16 +377,20 @@ def get_image_output(self):
 
     def get_digests(self):
         """
-        Returns a map of repositories to digests
+        Returns a map of images to their digests
         """
 
-        digests = {}  # repository -> digest
+        digests = {}  # repository -> digests
         for registry in self.workflow.push_conf.docker_registries:
             for image in self.workflow.tag_conf.images:
                 image_str = image.to_str()
                 if image_str in registry.digests:
-                    digest = self.select_digest(registry.digests[image_str])
-                    digests[image.to_str(registry=False)] = digest
+                    image_digests = registry.digests[image_str]
+                    if self.report_multiple_digests:
+                        digest_list = [image_digests.v1, image_digests.v2]
```

Comment:
Any chance either `.v1` or `.v2` is `None`?

Reply:
Only if the registry doesn't support v2, which I guess is a possibility we need to deal with.